tg/main.py

```diff
@@ -25,8 +25,8 @@ def interrupt_signal_handler(sig, frame):
 
     model = Model(tg)
     status_view = StatusView(stdscr)
-    msg_view = MsgView(stdscr, model.msgs, model, model.users)
```

Comment:
I found this approach more explicit and more flexible for future refactoring(removing `Model` class). Let's pass only what we need.

Reply:
What is your plan for refactoring? I didn't have any plans to do it in the nearest future. Can you describe it so that we can be on one page.